# Goal Request /moving_turtle/cmd_vel
geometry_msgs/Vector3 linear_pos 
geometry_msgs/Vector3 angular_pos

# Result - time
---
uint64 duration
 
---
# Feedback /moving_turtle/pose
float32 x_pos 
float32 y_pos
float32 theta_pos



